src/library/lakify.py

```diff
@@ -82,19 +78,20 @@ def process_hash_list(document_datasets):
             db.completeLakify(conn, doc_id)
 
         except (etree.XMLSyntaxError, etree.SerialisationError) as e:
-            logger.warning('Failed to extract activities to lake with hash {} and doc id {}'.format(file_hash, doc_id))
-            db.lakifyError(conn, doc_id, 'Failed to extract activities')
+            err_message = "Unknown error"
```

Comment:
Would be helpful to log this as an etree error instead of unknown, if possible.

Reply:
If `e.msg` is populated, then it's not "Unknown error" it will be what's in `e.msg`.